public/static/locales/en/donate.json

```diff
@@ -111,5 +111,6 @@
   "projectType": "Project Type",
   "conservation": "Conservation",
   "restoration": "Restoration",
-  "filters": "Filters"
-}
\ No newline at end of file
+  "filters": "Filters",
+  "perM2": "per m2"
```

Comment:
Maybe we should change the unit to m² here as well

Reply:
First I tried to do this with `<sup>2</sup>` in the translation resource, but then I thought we can also just enter the unicode representation `m²` into the resources.